backend/capellacollab/projects/toolmodels/modelsources/git/handler/handler.py

```diff
@@ -3,6 +3,9 @@
 
```

Comment:
I can't get it to run. The stacktrace is: 

```py
Traceback (most recent call last):
  File "/Users/moritzweber/Projects/github/capella-collab-manager/backend/capellacollab/projects/toolmodels/diagrams/routes.py", line 44, in get_diagram_metadata
    ) = await handler.get_file_from_repository_or_artifacts_as_json(
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/moritzweber/Projects/github/capella-collab-manager/backend/capellacollab/projects/toolmodels/modelsources/git/handler/handler.py", line 84, in get_file_from_repository_or_artifacts_as_json
    ) = await self.get_file_from_repository_or_artifacts(
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/moritzweber/Projects/github/capella-collab-manager/backend/capellacollab/projects/toolmodels/modelsources/git/handler/handler.py", line 110, in get_file_from_repository_or_artifacts
    job_id, last_updated = await self.get_last_job_run_id_for_git_model(
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/moritzweber/Projects/github/capella-collab-manager/backend/capellacollab/projects/toolmodels/modelsources/git/github/handler.py", line 30, in get_last_job_run_id_for_git_model
    jobs = self.get_last_pipeline_runs(project_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/moritzweber/Projects/github/capella-collab-manager/backend/capellacollab/projects/toolmodels/modelsources/git/github/handler.py", line 118, in get_last_pipeline_runs
    response.raise_for_status()
  File "/Users/moritzweber/Projects/github/capella-collab-manager/backend/.venv/lib/python3.11/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.github.com/repos/DSD-DBS/coffee-machine.git/actions/runs?branch=diagram-cache%2Fmain&per_page=20
```

Reply:
what puzzles me is why you are trying to get the actions from the `diagram-cache/main` branch? so it is correct that there will be no action found on that branch as there is no action for that branch.